private/rules/jvm_import.bzl

```diff
@@ -14,13 +14,10 @@ def _jvm_import_impl(ctx):
     if len(ctx.files.jars) != 1:
         fail("Please only specify one jar to import in the jars attribute.")
 
-    # With `bzlmod` enabled, workspace names end up being `~` separated. For the
-    # user-visible workspace name, we need the final part of the name
-    visible_name = ctx.label.workspace_name.rpartition("~")[2]
     label = "@{workspace_name}//{package}:{name}".format(
```

Comment:
in the interest of keeping names up to date, let's replace uses of `workspace_name` here with `repo_name` (or just `repo`)